cypress/integration/userCanSeeJournalistDashboard.feature.js

```diff
@@ -0,0 +1,10 @@
+describe('User can see journalist dashboard', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  describe('successfully', () => {
+    cy.get('[data-cy=create-article-btn]').should('exist')
+  })
+  
+})
```

Comment:
I don't see the relevance of this test. This functionality is tested in the cypress/integration/userCanCreateArticle.feature.js

Reply:
Test removed